src/videojs-hls.js

```diff
@@ -585,7 +621,8 @@ videojs.HlsHandler.prototype.selectPlaylist = function () {
     effectiveBitrate,
     sortedPlaylists = this.playlists.master.playlists.slice(),
     bandwidthPlaylists = [],
-    i = sortedPlaylists.length,
+    now = +new Date(),
```

Comment:
`Date.now()` is supported in all browsers we care to support. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now#Browser_compatibility
Not a blocker.